ControlRoomApplication/ControlRoomApplication/Database/Operations/DatabaseOperations.cs

```diff
@@ -61,6 +61,7 @@ public static void PopulateLocalDatabase()
             {
                 DateTime date = DateTime.Now.AddMinutes(5);
 
+                Appointment appt0 = new Appointment();
```

Comment:
You need to remember to notify others when you are going to be working in a file that they may be working in. This file's changes should have been completed in the DatabaseOperations branch to ensure no conflicts with any changes I may have made.

Reply:
I guess I'm still a little confused about how the changes could be completed in another branch if they depend on changes made in my branch. Should I make my changes, commit them, and then let the person know who has been working in the part of the project I want to change and let them pull from my branch and make the changes? Or should I just give them a heads up that I'll be changing that part of the code and clear the changes with them?

Reply:
Just give them the heads up that you're going to change a file they have/may have been working in, commit your changes and then switch branches and merge the work together. It's just that we want to avoid merging the work at the remote level in Development because then I'm the one merging the two different files and I won't necessarily know what work should/shouldn't be left out